helpers/theme.js

```diff
@@ -30,7 +31,7 @@ const instanceInstitute = () => {
 };
 
 const instanceCloudConsentNecessary = () => {
-	if (SC_THEME === 'n21' || SC_THEME === 'brb') {
+	if (!Configuration.get('FEATURE_CONSENT_NECESSARY')) {
```

Comment:
We can just return the value of the feature variable itself here, no?

Reply:
I would agree to that, why not just return the configuration value?

Reply:
I shifted this change and also some variable renaming to the follow-up ticket BC-2276